cypress/common/lessons.ts

```diff
@@ -283,33 +286,64 @@ export const lesson3 = ({
         problems: [],
       },
     ]);
-    runTests({
-      passing: 1,
-      failing: 0,
-      suites: [
-        {
-          basename: 'utils.test.ts',
-          dirname: 'src/__tests__',
-          passing: 1,
-          failing: 0,
-          tests: [
-            {
-              name: ['utils', baseTestName],
-              state: 'pass',
-            } as Test,
-          ].concat(
-            testName === undefined
-              ? []
-              : [
-                  {
-                    name: ['utils', testName],
-                    state: 'pass',
-                  } as Test,
-                ],
-          ),
-        },
-      ],
-    });
+    resetEditorTab({ path: `src/__tests__/utils.test.ts` });
+    if (chapter === 3 || chapter === 4 || chapter === 5 || chapter === 6) {
+      runTests({
+        passing: 1,
+        failing: 0,
+        suites: [
+          {
+            basename: 'utils.test.ts',
+            dirname: 'src/__tests__',
+            passing: 2,
+            failing: 0,
+            tests: [
+              {
+                name: ['utils', baseTestName],
+                state: 'pass',
+              } as Test,
+            ].concat(
+              testName === undefined
+                ? []
+                : [
+                    {
+                      name: ['utils', testName],
+                      state: 'pass',
+                    } as Test,
+                  ],
+            ),
+          },
+        ],
+      });
+    } else {
+      runTests({
+        passing: 1,
+        failing: 0,
+        suites: [
+          {
+            basename: 'utils.test.ts',
+            dirname: 'src/__tests__',
+            passing: 1,
```

Comment:
looks like this is the only field that's different right? Might be better to do something like `chapter === 3 || ... ? 2 : 1`